crypto/ml_dsa/ml_dsa_sign.c

```diff
@@ -47,8 +47,10 @@ static void signature_init(ML_DSA_SIG *sig,
  * @returns 1 on success and 0 on failure.
  */
 static int ml_dsa_sign_internal(const ML_DSA_KEY *priv, int msg_is_mu,
-                                const uint8_t *encoded_msg,
-                                size_t encoded_msg_len,
+                                const uint8_t *context_msg,
+                                size_t context_len,
```

Comment:
Can you be consistent with the names here please.. i.e context_msg and context_msg_len
 This is really an encoded ctx. (context_msg is not quite right).

Reply:
is it ok to rename these to enc_ctx and enc_ctx_len ?